developer/src/kmcmpdll/kmcmpdll.vcxproj.filters

```diff
@@ -98,9 +100,9 @@
     <ClCompile Include="..\..\global\vc\unicode.cpp" />
     <ClCompile Include="virtualcharkeys.cpp" />
     <ClCompile Include="..\..\global\vc\xstring.cpp" />
-    <ClCompile Include="..\..\ext\json-schema-validator\json-schema-draft4.json.cpp" />
-    <ClCompile Include="..\..\ext\json-schema-validator\json-uri.cpp" />
-    <ClCompile Include="..\..\ext\json-schema-validator\json-validator.cpp" />
+    <ClCompile Include="..\ext\json-schema-validator\json-schema-draft4.json.cpp" />
+    <ClCompile Include="..\ext\json-schema-validator\json-uri.cpp" />
+    <ClCompile Include="..\ext\json-schema-validator\json-validator.cpp" />
```

Comment:
Uh... glad we fixed the path now, at least.  But that means this should've been broken in a couple of the prior PRs?  And that didn't flag anything?

Reply:
.vcxproj.filters files are only used for grouping of files and presentation in the Project view in Visual Studio.